include/nffs/nffs.h

```diff
@@ -284,6 +289,29 @@ struct nffs_dir {
     struct nffs_dirent nd_dirent;
 };
 
+STATS_SECT_START(nffs_stats)
```

Comment:
this will now compile only on Zephyr

I'd either wrap everything in `#ifdef STATS_SECT_START` or create empty macros for non-Zephyr builds like we have for `STATS_INC` now. For Mynewt build (once it's finally changed to use this repo) we'll just add proper headers above later.

Reply:
@andrzej-kaczmarek I've don as you proposed.